src/main/java/core/basesyntax/JavaStreamApi.java

```diff
@@ -11,7 +12,8 @@
      * Вернуть сумму нечетных числел или 0, если таких несуществует</p>
      **/
     public Integer oddSum(List<Integer> numbers) {
-        return null;
+
+        return numbers.stream().filter(n -> n % 2 != 0).reduce(0,Integer::sum);
```

Comment:
```suggestion
        return numbers.stream()
        .filter(n -> n % 2 != 0)
        .reduce(0,Integer::sum);
```
давай писати стріми у відповідному стилі